src/server/builds/node.js

```diff
@@ -69,7 +69,8 @@ function algoliasearch(applicationID, apiKey, opts) {
 }
 
 algoliasearch.version = require('../../version.js');
-algoliasearch.ua = 'Algolia for Node.js (' + algoliasearch.version + ')';
+algoliasearch.ua = 'Node.js (' + process.versions.node + '); ';
+algoliasearch.ua += 'Algolia for Node.js (' + algoliasearch.version + ')';
```

Comment:
```suggestion
algoliasearch.ua = 'Node.js (' + process.versions.node + '); Algolia for Node.js (' + algoliasearch.version + ')';
```

Reply:
I prefer mutating it. It is easier to understand that we actually have two user-agents.

Reply:
We rarely use this kind of syntax `+=`. We avoid the mutation as much as possible. We can chain them in that case it won't be complexify that much. We can also invert the order makes more sense to have the `Algolia for ...` first.